packages/ember-glimmer/tests/utils/shared-conditional-tests.js

```diff
@@ -349,6 +351,59 @@ export const ObjectTestCases = {
     });
 
     this.assertText('T1T2F3');
+  },
+
+  ['@test setting the isTruthy property of an object proxy overrides its truthiness regardless of its content']() {
```

Comment:
This is likely testing observable behavior that was unintentional.  Likely it would have been better to have made this a symbol or meta.  I don't think we should block on this since it is just a port of an existing test.